saturn-r/Dockerfile

```diff
@@ -0,0 +1,21 @@
+
+ARG SATURNBASE_IMAGE
+FROM ${SATURNBASE_IMAGE}
+
+COPY environment.yml /tmp/environment.yml
+COPY postBuild /tmp/postBuild.sh
+COPY r-jupyter-kernel /tmp/r-jupyter-kernel.sh 
+# https://stat.ethz.ch/R-manual/R-devel/library/base/html/libPaths.html
+ENV R_LIBS=/usr/local/lib/R/
+
+RUN conda env update -n saturn --file /tmp/environment.yml && \
+    find ${CONDA_DIR} -type f,l -name '*.pyc' -delete && \
+    find ${CONDA_DIR} -type f,l -name '*.a' -delete && \
+    find ${CONDA_DIR} -type f,l -name '*.js.map' -delete
+    
+RUN /bin/bash -e -u -o pipefail /tmp/postBuild.sh && \
+    sudo chmod 755 /tmp/r-jupyter-kernel.sh
+
+RUN echo '' > ${CONDA_DIR}/envs/saturn/conda-meta/history
```

Comment:
could this command be moved up to after `find ${CONDA_DIR} -type f,l -name '*.js.map' -delete` and `&&`-ed into that block? Having fewer steps = fewer layers = smaller built image

Reply:
You mean the echo? I have no clue, that's just what I found in other default image builds, so I added it here.

Reply:
I think it can safely be moved up into that other block and `&&`-ed together with the other stuff. It's rewriting a history file to cut the image size a little bit.